pkg/disttask/framework/scheduler/scheduler.go

```diff
@@ -387,7 +393,7 @@ func (s *BaseScheduler) Rollback(ctx context.Context, task *proto.Task) error {
 
 	// We should cancel all subtasks before rolling back
 	for {
-		subtask, err := s.taskTable.GetFirstSubtaskInStates(s.id, task.ID, task.Step,
+		subtask, err := s.taskTable.GetFirstSubtaskInStates(ctx, s.id, task.ID, task.Step,
```

Comment:
use `rollbackCtx`?

Reply:
> use `rollbackCtx`?

Not neccessary? Since the ctx is used right before rollbackctx?But ok to change it.